File: Examples/Excel/VB/Standard Examples/ExcelExamples/FormMain.vb
Public Class FormMain

    Public Sub New()

        InitializeComponent()

        Me.Text = "NetOffice Excel Examples in Visual Basic"
        LoadExamples()

    End Sub

    Private Sub LoadExamples()

        LoadExample(New Example01())
        LoadExample(New Example02())
        LoadExample(New Example03())
        LoadExample(New Example04())
        LoadExample(New Example05())
        LoadExample(New Example06())
        LoadExample(New Example07())
        LoadExample(New Example08())
        LoadExample(New Example09())
        LoadExample(New Example10())
    End Sub

End Class

File: Examples/Excel/Automation/FormMain.cs
using System;
using ExampleBase;

namespace ExcelExamplesCS4
{
    public partial class FormMain : ExampleForm
    {
        public FormMain()
        {
            InitializeComponent();
            this.Text = "NetOffice Excel Examples in C#";
            LoadExamples();
        }

        private void LoadExamples()
        {
            LoadExample(new Example01());
            LoadExample(new Example02());
            LoadExample(new Example03());
            LoadExample(new Example04());
            LoadExample(new Example05());
            LoadExample(new Example06());
            LoadExample(new Example07());
            LoadExample(new Example08());
            LoadExample(new Example09());
            LoadExample(new Example10());
        }
    }
}
